django/db/models/options.py

```diff
@@ -825,3 +825,12 @@ def _property_names(self):
             if isinstance(attr, property):
                 names.append(name)
         return frozenset(names)
+
+    @cached_property
+    def returning(self):
+        return [
+            field for field in self._get_fields(
+                forward=True, reverse=False, include_parents=PROXY_PARENTS
+            )
+            if getattr(field, 'returning', False)
```

Comment:
you can drop the `False` to getattr here, `None` is also not true.

Reply:
Nope, this will raise an AttributeError in Py3 ;)
https://docs.python.org/3/library/functions.html#getattr